lib/extract.js

```diff
@@ -9,9 +9,10 @@ module.exports = function (request, options) {
   // The key holding token value in url or cookie defaults to token
   var urlKey = typeof options.urlKey === 'string' ? options.urlKey : 'token';
   var cookieKey = typeof options.cookieKey === 'string' ? options.cookieKey : 'token';
+  var allowUrlToken = typeof options.allowUrlToken === 'boolean' ? options.allowUrlToken : true;
   var auth;
 
-  if(request.query[urlKey]) { // tokens via url: https://github.com/dwyl/hapi-auth-jwt2/issues/19
+  if(allowUrlToken && request.query[urlKey]) { // tokens via url: https://github.com/dwyl/hapi-auth-jwt2/issues/19
```

Comment:
That was the whole idea to be able to turn the url key off. And I wanted it to default to true, so that it's not a breaking change. Another way would probably be to set the urlKey explicit to false.